modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -0,0 +1,106 @@
+
+* With this update, users can develop applications by verifying commit signatures for `ApplicationSet` CRD created using Git generators. Before this update, users could develop applications without the necessity of verifying commit signatures. link:https://issues.redhat.com/browse/GITOPS-3049[GITOPS-3049]
++
+[NOTE]
+====
+The signature verification does not function with the templated project field in `ApplicationSet` CRD Git Generator.
+====
+
+[id="fixed-issues-1-14-0_{context}"]
+== Fixed issues
+
+* Before this update, users could not execute a rollback for multi-source applications in the Argo CD Web UI or CLI. This update fixes the issues, and users can now carry out rollbacks for multi-source applications using the Argo CD Web UI or CLI. link:https://issues.redhat.com/browse/GITOPS-3996[GITOPS-3996]
+
+* Before this update, the `ApplicationSet` Controller role did not include the `list` and `watch` permissions for the `AppProject` resource, which resulted in errors during the application generation process. This update fixes the issue by granting necessary permissions to the `ApplicationSet` Controller role. link:https://issues.redhat.com/browse/GITOPS-5401[GITOPS-5401]
+
+* Before this update, Argo CD was unable to synchronize Persistent Volume resources on {OCP} v4.16 due to the introduction of the new `.status.lastPhaseTransitionTime` field in Kubernetes v1.29. This update fixes the issue by modifying the Kubernetes static scheme in Argo CD. link:https://issues.redhat.com/browse/GITOPS-5167[GITOPS-5167]
+
+* Before this update, the {gitops-title} Operator included a race condition where the OpenShift Service CA created the TLS secret before establishing the default TLS termination policy. This resulted in `Passthrough` being set as the default policy rather than `Reencrypt`. This update fixes this race condition by verifying whether the secret was created by the OpenShift Service CA, ensuring that `Reencrypt` is used as the correct default policy. link:https://issues.redhat.com/browse/GITOPS-4947[GITOPS-4947]
+
+* Before this update, adding a self-signed TLS certificate for the ApplicationSet GitLab SCM Provider did not function as intended, as the certificate failed to mount on the required volume. This update fixes the issue by correcting the volume mount path and providing documentation on the appropriate usage of this feature. Users are now required to ensure that the ConfigMap is designated as `argocd-appset-gitlab-scm-tls-certs-cm`. link:https://issues.redhat.com/browse/GITOPS-4801[GITOPS-4801]
++
+[NOTE]
+====
+The key for the TLS Certificate ConfigMap must be labeled 'cert', because this serves as the filename for the mounted certificate. Other key names will not be effective due to an upstream bug that will be resolved in a future release. The following is an example of a ConfigMap that can be utilized to mount your TLS certificate.
```

Comment:
```suggestion
The key for the TLS Certificate config map must be labeled 'cert' because this serves as the file name for the mounted certificate. Other key names will not be effective due to an upstream bug that will be resolved in a future release. The following is an example of a config map you can use to mount your TLS certificate.
```

Reply:
Fixed.